messages/run-command.md

```diff
@@ -50,10 +50,6 @@ If you want to preview the list of rules before you actually run them, use the `
 
     <%= config.bin %> <%= command.id %> --rule-selector eslint:getter-return --rule-selector no-inner-declarations
 
-- Specify three method starting points to be applied to the path-based recommended rules of the Salesforce Graph Engine, "sfge", while specifying the "./src" folder as the workspace to be used to build the source graph:
-
-    <%= config.bin %> <%= command.id %> --rule-selector sfge:Recommended --workspace ./src --path-start ./src/classes/Utils.cls#init --path-start ./src/classes/Helpers.cls#method1;method2
-
```

Comment:
There is no way to comment out code without it still showing up in the help with md files... so removing this for now.

Reply:
Question: Could you preserve it by declaring it as another message? E.g., `command.hidden-examples`?
I ask because I'm loathe to throw out approved text that we know we'll be using later.

Reply:
Great idea... will do that!

Reply:
Done.